google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -264,7 +331,7 @@ public boolean exists(String tableId, ByteString rowKey) {
    * }</pre>
    */
   public ApiFuture<Boolean> existsAsync(String tableId, String rowKey) {
-    return existsAsync(tableId, ByteString.copyFromUtf8(rowKey));
+    return existsAsync(tableId, rowKey, new ExistsOptions());
```

Comment:
why are we removing `ByteString.copyFromUtf8(rowKey)` here?

Reply:
I think this will call:
```
  public ApiFuture<Boolean> existsAsync(
      String tableId, String rowKey, @Nullable ExistsOptions existsOptions) {
    return existsAsync(tableId, ByteString.copyFromUtf8(rowKey), existsOptions);
```
which the string is converted there